config/src/chain.rs

```diff
@@ -97,12 +97,12 @@ impl From<Chain> for U256 {
 }
 
 impl TryFrom<Chain> for ethers_core::types::Chain {
-    type Error = ParseChainError;
+    type Error = String;
 
     fn try_from(chain: Chain) -> Result<Self, Self::Error> {
         match chain {
             Chain::Named(chain) => Ok(chain),
-            Chain::Id(id) => id.try_into(),
+            Chain::Id(id) => ethers_core::types::Chain::try_from(id).map_err(|err| err.to_string()),
```

Comment:
https://github.com/foundry-rs/foundry/pull/3937/commits/322ba09a4cee500e77c49ef488ecb7037c909b53

```suggestion
            Chain::Id(id) => id.try_into(),
```